src/utils/answerLinksUtil.js

```diff
@@ -0,0 +1,9 @@
+import { LINKS_DIV } from '../constants.js';
+import { createLinkElement } from '../views/linkView.js';
+
+export const answerLinks = (currentQuestion) => {
```

Comment:
I think this function's responsibility is to append links. Should we rename the function as `appendLinks`? What do you think?

Reply:
sure